packages/noya-designsystem/src/components/DropdownMenu.tsx

```diff
@@ -134,6 +135,9 @@ function DropdownMenuRoot<T extends string>({
     [isNested, items, onSelect, shouldBindKeyboardShortcuts],
   );
 
+  const style = isScroll
```

Comment:
I like this maxHeight approach cause it's simple, and we could make the maxHeight a prop. 

I think it'd be ideal if the menu extended to the bottom of the screen, minus some padding. How difficult do you think that'd be? It it's hard it might not be worth it.